tezos/caip122.md

```diff
@@ -0,0 +1,74 @@
+---
+namespace-identifier: tezos-caip122
+title: Tezos Namespace - SIWx
+author: Qibing Li (@QibingLee)
+discussions-to: TBA
+status: Draft
+type: Standard
+created: 2022-12-06
+updated: 2022-12-06
+requires: ["CAIP-122", "CAIP-2", "CAIP-10"]
+---
+
+## CAIP-122
```

Comment:
for Ed25519, this caip-122 for this namespace looks good, since same message format and pretty standard Ed25519 stuff, but do you want to define it for Secp256k1 and P256 as well?

Reply:
Actually, we can also define the Secp256k1 and P256, so did:tz1, did:tz2, or did:tz3 can be used. Then generated signature and its public_key should be grouped together, to ensure verification without any external APIs.

Reply:
I have committed new changes to the Tezos namespace. Please check it for us